docs/concepts.md

```diff
@@ -6,22 +6,34 @@ menu: docs
 
 # Concepts
 
+## Application
+
+Any kind of piece of software that you would want to run in your Kubernetes cluster. It can be as simple as cleanup job or as complex as distributed system like Kafka.
```

Comment:
this sentence is awkward to read... in particular "Any kind of piece of software".. perhaps 
```suggestion
Any kind of software that you would want to run in your Kubernetes cluster. It can be as simple as cleanup job or as complex as distributed system like Kafka.
```